src/content/docs/style-guide/ui-writing/button-labels.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Button labels
+---
+
+A mislabeled or misleading button can cause confusion when users go to take actions in our UI. We need button labels that are clear, accurate, and descriptive.
+
+## Basics
+
+-   Make the label match what is actually going to happen when users click the button.
+-   Use multiple words if one isn't clear enough.
+-   Avoid end punctuation, especially exclamation marks.
+-   Keep buttons in sentence case.
+
+| Location              | Do                 | Don't       |
+| --------------------- | ------------------ | ----------- |
+| Chart editing dialog  | Save chart         | OK, Save    |
+| Dashboards main page  | Create a dashboard | Add, Create |
+| Discovery center tour | Take the tour      | Tour, Go!   |
+| NRQL query            | Run                | Apply       |
+
+# Exceptions to the rule
+
+We're deliberately blending the concepts of search and filter in our various filter bars in New Relic One, specifically for distributed tracing, maps, logs, and the shared filter component that appears in the chart builder. We don't want to call out one or the other concept, so we're avoiding those specific terms.
+
+In these cases, we're using descriptive language to set up the actions, and then a generic "Apply" button to apply your changes and see your results.
+
+NOTE: This does not apply to the "Advanced (NRQL)" tab.
```

Comment:
@mmfred down the road you might add a <Callout variant="tip"> tag around this note.